lms/static/scripts/frontend_apps/api-types.ts

```diff
@@ -30,6 +34,10 @@ export type File = {
   children?: File[];
 };
 
+export type Page = Resource & {
```

Comment:
This could use a comment to indicate what a "Page" is. This could be something along the lines of "Canvas Pages or similar resources."